komodo/data/jenkins.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
What is the purpose of this file?

Reply:
You tell me, it came from `komodo-releases`. https://github.com/equinor/komodo-releases/blob/master/jenkins.sh